.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
```

Comment:
I think #1233 may be the cause of some of the slowness locally and on Travis-CI. I'd wager that being more aggressive with globs and copying directories instead of individual files would speed things up, as well as not running `babel` on each individual file. We could maybe start with concatenating a bunch of the client/server files into bundles and then only need to run babel 1-2 times.